docs/pages/machine-id/faq.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Machine ID FAQ
+description: Frequently asked questions about Teleport Machine ID
+---
+
+## Which Teleport features are currently supported by Machine ID?
+
+Broadly, Machine ID currently supports the following Teleport features:
```

Comment:
What does it mean to say that Machine ID supports a Teleport feature? I think it would help clarify things to add a sentence explaining this. We can also link to the Machine ID with Databases guide.

Reply:
That's fair, I've attempted to clarify this a bit. I've also changed the database access link to point to the database guide (assuming that's what you meant)